README.md

```diff
@@ -27,23 +34,21 @@ from happening. Most logging happens at `INFO`, which is the default setting.
 
 ## Local Development
 
-All of our server code is written using [Flask](https://flask.palletsprojects.com/en/2.3.x/).
-
-The Flask web service exists within `/src`. The `__init__.py` is the
-entry point for the app. The other files provide the routes.
-
-Other related Python code that implement features are within `/lib`.
+* Install docker 
+  * If you are on WSL, installing docker on the linux system wouldn't work as linux itself is running in a container. Install docker desktop instead following these instructions: https://learn.microsoft.com/en-us/windows/wsl/tutorials/wsl-containers
 
-To build the app, use `docker compose build`.
+* To build the app, use `docker compose build`.
 You will need to rebuild when you change the source.
 
-To run the app locally, use `docker compose up` from the repo root.
-
-This will run a webserver accessible at <http://localhost:5000>.
+* To run the app locally, use `docker compose up` from the repo root.
 
 **Note**: You need to provide the API keys in the `config.txt` file
 before the service runs. See the above "Configuration" section.
 
+This will run a webserver accessible at <http://localhost>.
```

Comment:
The default configuration in the docker file was for port 80, I noticed something similar when testing as well in my local instance. Please let me know if I'm missing something here.

Reply:
I can confirm that the localhost port is now 80. @cat5inthecradle or @wilkie does this seem like a regression?

Reply:
I'm guessing its defined here https://github.com/code-dot-org/aiproxy/blob/7706cf3623eb983216a8d8291621ca0ba14e98fe/Dockerfile#L12 and the port was updated by this commit https://github.com/code-dot-org/aiproxy/commit/ab8a5708c51ed637e39993959bd011f38f774c5a, but I'm not able to track down the corresponding PR by searching for the SHA hash in PRs

Reply:
@cat5inthecradle did you commit this directly to the main branch, or did someone secretly migrate this repo to LFS too?

 seems intentional enough and I don't see a reason not to keep it on port 80.